domains/thepersonalsite.json

```diff
@@ -4,6 +4,9 @@
         "email": "jessehoekema@icloud.com"
     },
     "record": {
-        "URL": "https://jessehoekema.github.io/tps/"
+        "A": "185.199.108.153"
```

Comment:
A record should be provided in a single array. Duplicate properties are not valid JSON.